app/models/setting/general.rb

```diff
@@ -21,7 +21,7 @@ def self.default_settings
       self.set('proxy_request_timeout', N_("Open and read timeout for HTTP requests from Foreman to Smart Proxy (in seconds)"), 60, N_('Smart Proxy request timeout')),
       self.set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text')),
       self.set('host_power_status', N_("Show power status on host index page. This feature calls to compute resource providers which may lead to decreased performance on host listing page."), true, N_('Show host power status')),
-      self.set('http_proxy', N_('Sets a proxy for all outgoing HTTP connections.'), nil, N_('HTTP(S) proxy')),
+      self.set('http_proxy', N_('Sets a proxy for all outgoing HTTP connections (established via foreman / a foreman plugin). System-wide proxies must be configured at operating system level.'), nil, N_('HTTP(S) proxy')),
```

Comment:
I think "Sets a proxy for all outgoing HTTP connections from Foreman. System-wide proxies must be configured at operating system level." reads a little better.